electronics_lib/PassiveCapacitor.py

```diff
@@ -1,10 +1,9 @@
 from typing import NamedTuple
 import math
-
 from electronics_abstract_parts import *
 from electronics_abstract_parts.Categories import DummyDevice
 from .DigikeyTable import *
-
+from .TableDeratingCapacitor import *
 
```

Comment:
I actually haven't touched select_capacitor function of SmtCeramicCapacitor because we are not working with at the moment. Plus, I have been using it as a reference. 
 
Refactoring the function code should be very easy and straight forward.

Reply:
Yeah, so the point of refactoring is to not have duplicated code - you're trying to pull out (factor out) the common code of both cases into a shared function, instead of having two duplicates.

The reasoning here is that if we want to make changes for all capacitors, we only need to change it in a single places, instead of hunting around a bunch of copies.

Reply:
I think I should ask this from you during the meeting today.